orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -43,13 +54,20 @@ class Outputs:
     def __init__(self):
         self.data = None
         box = gui.vBox(self.controlArea, 'Differencing')
+
+        gui.comboBox(box, self, 'chosen_operation', 
+                        orientation=Qt.Horizontal,
+                        items=['Difference','Quotient', 'Percentage change'], label='Compute :',
```

Comment:
Someone should maybe add flake8 check as a first line defense on Travis? Also, what's with the extra label space? :smiley: